server-phoenix/lib/helios_web/controllers/web_hooks/slack_controller.ex

```diff
@@ -1,12 +1,76 @@
 defmodule HeliosWeb.WebHooks.SlackController do
   use HeliosWeb, :controller
-  alias Helios.{Repo, Event}
+  require Logger
+  alias Helios.{Repo, Event, SlackChannelNames}
+
+  defp insert_channel_id(channel_id) do
+    unless SlackChannelNames
+           |> SlackChannelNames.with_id(channel_id)
+           |> Repo.exists?() do
+      try do
+        body = %{
+          "channel" => channel_id
+        }
+
+        request_body = URI.encode_query(body)
+
+        headers = [
+          {"Accept", "application/json"},
+          {"Authorization",
+           "Bearer #{Application.get_env(:helios, HeliosWeb.Endpoint)[:slack_bearer_token]}"},
```

Comment:
This line is a mouthful, both here and in `handle/2`. Might be worth pulling out a `defp slack_bearer_token()` to improve legibility.